config/foreman-tasks.yaml.example

```diff
@@ -9,6 +9,18 @@
 #     :action:
 #       :enabled: true
 
+# Task backup configuration can be changed by uncommenting the backup
+# section
+#
+#   :backup:
+#
+# Whether to back up tasks when they are removed
+#
+#     :backup_deleted_tasks: true
+#
+# Where to put the tasks which were backed up
+#
+#     :backup_dir: /var/lib/foreman/tasks-backup
```

Comment:
I think this was not commented on purpose, as it's being used in the packaging https://github.com/mbacovsky/foreman-packaging/blob/22d5cb766cf88646bdd72cf2bf433718c0e4c6f9/rubygem-foreman-tasks/rubygem-foreman-tasks.spec#L97

Reply:
Oh, I see